### PR TITLE
updating docs for some window functions and json table

### PR DESCRIPTION
**Pull request by jycor**
i didn't forget this time :)

to be merged when these make it to main
 - https://github.com/dolthub/go-mysql-server/pull/1190
 - https://github.com/dolthub/go-mysql-server/pull/1142
 - https://github.com/dolthub/go-mysql-server/pull/1199

### CODE REVIEW

File: content/reference/sql/sql-support/expressions-functions-operators.md [hunk 1]
```diff
@@ -54,449 +54,449 @@ title: "Expressions, Functions, and Operators"
...
+| `AVG()`                           | ✅         |                                                                   |
+| `BENCHMARK()`                     | ❌         |                                                                   |
+| `BETWEEN ... AND ...`             | ✅         |                                                                   |
+| `BIN()`                           | ✅         |                                                                   |
+| `BIN_TO_UUID()`                   | ✅         |                                                                   |
+| `BIT_AND()`                       | ✅         |                                                                   |
+| `BIT_COUNT()`                     | ❌         |                                                                   |
+| `BIT_LENGTH()`                    | ✅         |                                                                   |
+| `BIT_OR()`                        | ✅         | `\                                                                |` is supported                                                                              |
```
Comment (timsehn) [inline content/reference/sql/sql-support/expressions-functions-operators.md]: This is weird. You sure it's gonna render fine?

**Review by timsehn — APPROVED** [review]
Just the one comment on `BIT_OR()`  formatting looking weird.

File: content/reference/sql/sql-support/expressions-functions-operators.md (hunk 1 above)
Comment (jycor) [reply content/reference/sql/sql-support/expressions-functions-operators.md]: not sure how that got there, will fix

File: content/reference/sql/sql-support/expressions-functions-operators.md [hunk 2]
```diff
@@ -54,449 +54,449 @@ title: "Expressions, Functions, and Operators"
...
+| `VERSION()`                       | ✅         |                                                                   |
+| `WAIT_FOR_EXECUTED_GTID_SET()`    | ❌         |                                                                   |
+| `WEEK()`                          | ✅         |                                                                   |
+| `WEEKDAY()`                       | ✅         |                                                                   |
+| `WEEKOFYEAR()`                    | ✅         |                                                                   |
+| `WEIGHT_STRING()`                 | ❌         |                                                                   |
+| `YEAR()`                          | ✅         |                                                                   |
+| `YEARWEEK()`                      | ✅         |                                                                   |
 
 ## Aggregate Functions
```
Comment (timsehn) [inline content/reference/sql/sql-support/expressions-functions-operators.md:501]: Check this section. These functions are listed twice :-(

**Review by timsehn — CHANGES_REQUESTED** [review]

**Review by timsehn — APPROVED** [review]
LGTM. Ship it.

**Pull request merged**